stdlib/Serialization/src/Serialization.jl

```diff
@@ -391,16 +391,15 @@ function serialize(s::AbstractSerializer, meth::Method)
     serialize(s, meth.line)
     serialize(s, meth.sig)
     serialize(s, meth.slot_syms)
-    serialize(s, meth.ambig)
     serialize(s, meth.nargs)
     serialize(s, meth.isva)
     if isdefined(meth, :source)
-        serialize(s, uncompressed_ast(meth, meth.source))
+        serialize(s, Base._uncompressed_ast(meth, meth.source))
     else
         serialize(s, nothing)
     end
     if isdefined(meth, :generator)
-        serialize(s, uncompressed_ast(meth, meth.generator.inferred))
+        serialize(s, Base._uncompressed_ast(meth, meth.generator.inferred)) # XXX: what was this supposed to do?
```

Comment:
Looks like this code was wrong; `generator.inferred` doesn't exist any more. It should probably just be `meth.generator`.

Reply:
Perhaps so? Needs an issue opened (I added the comment in hopes that someone would eventually remember to do so).